jekyll/_cci2/ios-migrating-from-1-2.md

```diff
@@ -0,0 +1,386 @@
+jobs:
+  build-and-deploy:
+    steps:
+      ...
+      - run:
+          name: Run tests
+          command: bundle exec fastlane scan
+          environment:
+            SCAN_DEVICE: iPhone 6
+            SCAN_SCHEME: WebTests
+```
+
+You can replace the `bundle exec fastlane scan` command with your custom
+test command, and change the environment variables passed into it. If
+your test command spans multiple lines, you can include multiple
+commands in a single step:
+
+```
+jobs:
+  build-and-deploy:
+    steps:
+      ...
+      - run:
+          name: Run tests
+          command: |
+            make build
+            make test
+```
+
+### Storing artifacts, test results and diagnostic files
+
+In 2.0 we don't automatically collect artifacts in your builds anymore,
+so if your build is generating files that you would like to access via
+the UI later on, you'll need to explicitly collect those files via the
+`store_artifacts` step.
+
+The same is true for the XML test results: for us to be able to show
+them in the UI, you'll need to add a `store_test_results` step into your
+config.
+
+If you would like to save any kind of logs into the artifacts, you'll
+need to add that to the config too, for example like this:
+
+```
+jobs:
+  build-and-deploy:
+    steps:
+      ...
+      - store_test_results:
+          path: test_output/report.junit
+      - store_artifacts:
+          path: /tmp/test-results
+          # Destination is the name of the item in the Artifacts tab
+          # that will contain the paths specified in this step.
+          destination: scan-test-results
+      - store_artifacts:
+          path: ~/Library/Logs/scan
+          destination: scan-logs
+```
+
+You will find more details about these steps in the
+[Artifacts doc]({{ site.baseurl}}/2.0/artifacts/) and the
+[Test Metadata doc]({{ site.baseurl}}/2.0/collect-test-data/).
+
+### Deployment via Workflows
+
+With the availability of Workflows in 2.0, we suggest extracting all the
+commands related to the deployment of the app into its own job:
+
+```
+jobs:
+  ...
+  deploy:
+    macos:
+      xcode:
+        version: 8.3.3
+
+    steps:
+      - checkout
+
+      # Set up code signing via Fastlane Match.
+      - run:
+          name: Set up code signing
+          command: fastlane match development --readonly
+
+      # Build the release version of the app.
+      - run:
+          name: Build IPA
+          command: bundle exec fastlane gym
+
+      # Store the IPA file in the build artifacts
+      - store_artifacts:
+          path: ./MyApp.ipa
+          destination: ipa
+
+      # Deploy!
+      - run:
+          name: Deploy to App Store
+          command: bundle exec fastlane spaceship
+```
+
+In the deployment job we also specify an Xcode version, and then add the
+commands that will produce the release version of the app, store it in
+the artifacts, and submit it to App Store.
+
+We will also need to add a Workflows section into the config to specify
```

Comment:
change 'We will also need to add a Workflows...' to

The following snippet adds a Workflows section to the `.circleci/config.yml` file to specify the conditions that run the deployment job: